### DISCUSSION TITLE
[Cosmetic]: Known Issues link doesn't work

### DISCUSSION THREAD

**Discussion by puterfixer** [mbentley/docker-omada-controller#480] [Help] [answered]
### Controller Version

5.13.30.8

### Describe Your Issue or Question

Hey Matt,

Not really an Omada issue, just cosmetic for the project page on Github. :)

Basically I wanted to see if there are still significant outstanding issues with 5.14 before I upgrade, and found out that the Known Bugs link on Github doesn't work anymore (https://hub.docker.com/r/mbentley/omada-controller/KNOWN_ISSUES.md).

The file is reachable in the master branch on this link (https://github.com/mbentley/docker-omada-controller/blob/master/KNOWN_ISSUES.md).

Probably Github changed something and some of the old links to MD files return 404s.

Thanks!

### Expected Behavior

Expected to click a link and find the right contents, not a 404. Github, that's not very elegant of you...

### Steps to Reproduce

1. Open the project page https://hub.docker.com/r/mbentley/omada-controller/
2. Open the "README for v3 and v4" above the TOC, or "Known Issues readme" in Image Tags section, or "Known Issues" link at the very bottom.
3. Confront the disappointment of Github's inability to gracefully redirect old links when they change stuff in the backend, and just throw the towel.

### How You're Launching the Container

```plain
n/a for the issue :)
```

### Container Logs

```plain
n/a for the issue :)
```

**Comment by mbentley** [comment, marked as answer]
Ah yeah, that's because the links on GitHub are relative so they work but on the published README on Docker Hub, the page doesn't exist.  What I should probably do is have a unique README for Docker Hub and publish that which will mostly link to the README on GitHub because like none of the links work in the Table of Contents and I am sure there are tons of other links that do not work seeing as I have several other sub-pages of README because the README is just so large for the repo at this point.

Thanks for pointing that out, I haven't looked at the Docker Hub page in a long time.  I created #481 to track this being fixed.

**Reply by puterfixer** [reply]
You rock! Also, this is far too early in the day to go through bug reports... :D

**Reply by mbentley** [reply]
This should be fixed now as a mostly simple page pointing to the GitHub repo.
